.github/workflows/check.yml

```diff
@@ -29,3 +29,21 @@ jobs:
 
       - name: Build the bootloader (GNU, riscv64)
         run: ./bootstrap && ./configure TOOLCHAIN_FOR_TARGET=riscv64-linux-gnu --enable-werror --enable-uefi-riscv64 && make all && make maintainer-clean
+
+  build_nix:
```

Comment:
One last thing: I think this should be set to allow failures. Otherwise, I'm concerned that changes to the repo result in frequent errors because `bootstrappedSrcHash` will be invalid.

This can eventually be solved of the todo in bootstrappedSrc is addressed

Reply:
Problem fixed. Will push soon!